### ISSUE TITLE
Use result names in docstrings for result names in generated stubs

### ISSUE THREAD

**Issue by lars-reimann** [Safe-DS/Stub-Generator#100]
### Is your feature request related to a problem?

Results currently always have names `result1`, `result2`, etc. Frequently, however, the docstrings contain proper names for the results.

### Desired solution

Use result names in docstrings for result names in generated stubs.

**Comment by lars-reimann** [comment]
@Masara Reopening this, since it sometimes does not work:

1. Generate stubs for the library.
2. Check e.g. `builtins\out\safeds\ml\classical\regression\_linear_regression\_linear_regression.sdsstub`:

```
    /**
     * Create a copy of this regressor and fit it with the given training data.
     *
     * This regressor is not modified.
     *
     * @param trainingSet The training data containing the feature and target vectors.
     *
     * @result fittedRegressor The fitted regressor.
     */
    @Pure
    fun fit(
        @PythonName("training_set") trainingSet: TaggedTable
    ) -> result1: LinearRegressionRegressor
```

The result name in the docstring is correct, but the result is still called `result1`.

**Comment by Masara** [comment]
@lars-reimann should now be fixed with PR #108

**Comment by lars-reimann** [comment]
> @lars-reimann should now be fixed with PR #108

Works again now, thanks.

**Comment by lars-reimann** [comment]
There are still some instances where the result is called `result1` even though there's a name in the docstring. Generate stubs for https://github.com/Safe-DS/Library/commit/46f54af61f860ee3790685c330accde1ae4d033f and search globally for `result1`. Examples:

* `LogisticRegressionRegressor.predict`
* `AdaBoostRegressor.predict`
* `LabelEncoder.transform`
* `LabelEncoder.reverseTransform`

Maybe it's broken for overriding methods?

**Comment by Masara** [comment]
I'll fix this once #112 is merged.
